WordPress/src/main/java/org/wordpress/android/ui/sitecreation/misc/SiteCreationTracker.kt

```diff
@@ -271,4 +274,31 @@ class SiteCreationTracker @Inject constructor(
                 mapOf(VERTICAL_SLUG.key to verticalSlug)
         )
     }
+
+    // region Site Name
+
+    fun trackSiteNameViewed() {
+        tracker.track(AnalyticsTracker.Stat.ENHANCED_SITE_CREATION_SITE_NAME_VIEWED)
+    }
+
+    fun trackSiteNameCanceled() {
+        tracker.track(AnalyticsTracker.Stat.ENHANCED_SITE_CREATION_SITE_NAME_CANCELED)
+    }
+
+    fun trackSiteNameSkipped() {
+        tracker.track(AnalyticsTracker.Stat.ENHANCED_SITE_CREATION_SITE_NAME_SKIPPED)
+    }
+
+    fun trackSiteNameEntered() {
+        tracker.track(AnalyticsTracker.Stat.ENHANCED_SITE_CREATION_SITE_NAME_ENTERED)
+    }
+
+    fun trackSiteIntentQuestionExperimentVariation() {
+        tracker.track(
+                AnalyticsTracker.Stat.ENHANCED_SITE_CREATION_INTENT_QUESTION_EXPERIMENT,
+                mapOf(VARIATION.key to siteNameABExperiment.getVariation())
+        )
+    }
```

Comment:
I assume you meant `s/siteIntent/siteName/` here, so addressed with: https://github.com/wordpress-mobile/WordPress-Android/commit/02c8c5049701aff67ca3a36d4b26d5fec5d9d157 in the collaborative branch. Mentioning here in case that is not used.

Reply:
Indeed, thank you! 🙇

So much for copy-pasta 😅